mips32: Build fails with error: 'ODD_SPREG' was not declared in this scope

Trying to build revision 324dfb828cec09f7638b48b6e25e4007b45b9cbc on mips32 (little-endian) fails with:

```
g++  -fno-PIE -c  -I../../gcc/../zlib -g -O2 -DIN_GCC     -fno-exceptions -fno-rtti -fasynchronous-unwind-tables -W -Wall -Wno-narrowing -Wwrite-strings -Wcast-qual -Wno-error=format-diag -Wmissing-format-attribute -Woverloaded-virtual -pedantic -Wno-long-long -Wno-variadic-macros -Wno-overlength-strings -fno-common  -DHAVE_CONFIG_H -I. -I. -I../../gcc -I../../gcc/. -I../../gcc/../include -I../../gcc/../libcpp/include -I../../gcc/../libcody  -I../../gcc/../libdecnumber -I../../gcc/../libdecnumber/dpd -I../libdecnumber -I../../gcc/../libbacktrace   -o gcov.o -MT gcov.o -MMD -MP -MF ./.deps/gcov.TPo ../../gcc/gcov.c
../../gcc/config/mips/mips-rust.c: In function 'void mips_rust_target_cpu_info()':
../../gcc/config/mips/mips-rust.c:207:11: error: 'ODD_SPREG' was not declared in this scope
  207 |     if (!(ODD_SPREG))
      |           ^~~~~~~~~
```

--- a/gcc/config/mips/mips-rust.c
+++ b/gcc/config/mips/mips-rust.c
@@ -204,7 +204,7 @@ void mips_rust_target_cpu_info(void) {
         rust_add_target_info("target_feature", "single-float");
     if (TARGET_SOFT_FLOAT_ABI)
         rust_add_target_info("target_feature", "soft-float");
-    if (!(ODD_SPREG))
+    if (!TARGET_ODD_SPREG)
         rust_add_target_info("target_feature", "nooddspreg");
     // TODO: find if vfpu (vector FPU instructions) are supported by gcc at all
     // TODO: ensure below switch variable and whatever works

